### PR TITLE
Mes 1104 record showme

### PR DESCRIPTION
**Pull request by Jason-Pullen**
mes-1104 record the show me question
with validation

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [ ] commits are squashed

## Sign off process checklist:

- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github
- [ ] Any new reusable component/widget added to component library on Confluence

- [x] Screenshot(s) captured on the physical device (if applicable)
- [ ] Tested by QA
- [ ] PO's approval

Screenshot 2019-04-16 at 13 36 30
Screenshot 2019-04-16 at 13 35 21
Screenshot 2019-04-16 at 13 35 15

### CODE REVIEW

File: src/modules/tests/vehicle-checks/__tests__/vehicle-checks.reducer.spec.ts [hunk 1]
```diff
@@ -29,4 +34,33 @@ describe('vehicle checks reducer', () => {
...
+      showMeQuestionDescription: 'desc',
+      showMeQuestionShortName: 'name',
+    };
+
+    const result = vehicleChecksReducer({}, new ShowMeQuestionSelected(newQuestionPayload));
+    expect(result.showMeQuestionCode).toBe('S1');
+    expect(result.showMeQuestionDescription).toBe('desc');
+  });
+
+  it('it should update the show me question details', () => {
```
Comment (rossfauldsbjss) [inline src/modules/tests/vehicle-checks/__tests__/vehicle-checks.reducer.spec.ts]: Our tests never use it in the it, start with should. would remove it in the text for all 
(that sounds clear as mud - happy to clarify)

Comment (Jason-Pullen) [reply src/modules/tests/vehicle-checks/__tests__/vehicle-checks.reducer.spec.ts]: clear as mud..my bad..

**Comment by buzzub** [comment]
Think this is probably related to an earlier PR, but on the validation for show me, shouldn't the text underneath read "Select the show me question" rather than "Enter the show me question" as it's not an input field?

Also, on the same screenshot, the red bar on the left should be red but it isn't.

File: src/pages/office/office.scss [hunk 2]
```diff
@@ -1,5 +1,9 @@
 page-office {
     ion-content {
+        // background-color: map-get($colors, "mes-white");
```
Comment (johnmorleybjss) [inline src/pages/office/office.scss]: Comment.

**Pull request merged**